### PR TITLE
Fix broken image issue

### PR DESCRIPTION
**Pull request by mghazanfar**

### CODE REVIEW

File: src/common/components/profile-preview/index.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ export const ProfilePreview = ({username, global, onClose, activeUser, ...props}
                         <div className={`rounded-circle mb-3 profile-img-container ${profile && profile.profile.profile_image ? "" : "no-image"}`}>
                             {loading ? <Skeleton className="profile-img rounded-circle" /> : profile && 
                             <Link to={`/@${username}`} onClick={(e) => onClose(e, true)}>
-                                <img src={`https://images.ecency.com/u/${username}/avatar/medium`} alt="profile-image" className="profile-img rounded-circle bg-primary" loading="lazy"/>
+                                <img src={`https://images.ecency.com/u/${username}/avatar/medium`} alt="img" className="profile-img rounded-circle bg-primary" loading="lazy"/>
```
Comment (feruzm) [inline src/common/components/profile-preview/index.tsx:105]: This should've use `webp` check, if supports should use webp format.

Comment (feruzm) [reply src/common/components/profile-preview/index.tsx:105]: @mghazanfar ^

Comment (mghazanfar) [reply src/common/components/profile-preview/index.tsx]: @feruzm It worked. Pushed.

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**